hcl/hcl.go

```diff
@@ -321,6 +321,21 @@ func (p *TerramateParser) MinimalParse() error {
 	return errs.AsError()
 }
 
+// Imports returns all import blocks parsed.
+func (p *TerramateParser) Imports() (ast.Blocks, error) {
+	errs := errors.L()
+	imports := ast.Blocks{}
+
+	for _, importBlock := range filterBlocksByType("import", p.Blocks) {
+		err := validateImportBlock(importBlock)
+		errs.Append(err)
+		if err == nil {
+			imports = append(imports, importBlock)
+		}
+	}
+	return imports, errs.AsError()
```

Comment:
```suggestion
	if err := errs.AsError(); err != nil {
		return err
	}
	return imports, nil
```

Not sure which path we're gonna have here but either way some changes are needed.
The "suggestion" here is to just follow our convention of returning either the value or error. But the in the original implementation, we were continuing parsing other imports in case of errors, to return the maximum amount of diagnostics possible. If we continue with this idea, then you can add a comment here explaining why we are returning both the value and errors, and then the code using this function needs to change to not return in case of error.

In my opinion, it'd be better if we continue parsing/validating other imports.

Reply:
I was just lazy with eh extra if, the caller should never use the returned value if the error is not nil anyway, but adding the if is more explicit.

Reply:
ok but now we don't send diagnostics for the other imports if one of them is invalid (missing source, source is not string, etc), it's going to clean the problems of other imported files...